src/library.js

```diff
@@ -3244,6 +3244,7 @@ LibraryManager.library = {
   },
 
   emscripten_asm_const_int__sig: 'iiii',
+  emscripten_asm_const_int__deps: ['$readAsmConstArgs'],
   emscripten_asm_const_int: function(code, sigPtr, argbuf) {
 #if RELOCATABLE
     code -= {{{ GLOBAL_BASE }}};
```

Comment:
lower down this is also missing on `$mainThreadEM_ASM`, and I think it might break without adding it given this PR's changes to emscripten.py

Reply:
ah yes, thanks.